src/pages/NewAndEditVRS.page

```diff
@@ -28,41 +28,104 @@
-			  </apex:actionRegion>			    
-			</apex:pageBlockSection>
-        </apex:pageBlock>	
-	</apex:form> 
+<apex:page standardController="Volunteer_Recurrence_Schedule__c"
+    extensions="VOL_CTRL_NewAndEditVRS"
+    title="{!$ObjectType.Volunteer_Recurrence_Schedule__c.Label}"
+    tabStyle="Volunteer_Recurrence_Schedule__c"
+    docType="html-5.0"
+    standardStylesheets="false">
+
+    <apex:stylesheet value="{!URLFOR($Resource.sfdo213, '/assets/styles/salesforce-lightning-design-system-vf.min.css')}" />
+    <apex:includeScript value="{!URLFOR($Resource.jQueryFiles, 'js/jquery-1.4.4.min.js')}"/>
+    <script>
+        function fixMultiPicklists() {
+            $('.multiSelectPicklistTable').hide();
+            $('.multipicklistSLDSFixup').show();
+        }
+        var previousOnload = window.onload;
+        window.onload = function() {
+            if (previousOnload) {
+                previousOnload();
+            }
+            fixMultiPicklists();
+            document.getElementById('{!$Component.vfForm.slJRS}').focus();
+        }
+    </script>
+
+    <apex:form id="vfForm">
+        <div class="sfdo213">
+            <!-- PAGE SPINNER -->
+            <div class="slds-spinner_container" style="z-index:999;position:fixed;display:none;" id="divLoading">
+                <div class="slds-spinner--brand slds-spinner slds-spinner--medium" style="position:fixed;top:30%;" aria-hidden="false" role="alert">
+                    <div class="slds-spinner__dot-a"></div>
+                    <div class="slds-spinner__dot-b"></div>
+                </div>
+            </div>
+            <!-- PAGE HEADER -->
+            <div class="slds-page-header" role="banner">
+                <div class="slds-grid">
+                    <div class="slds-col">
+                        <h1 class="slds-text-heading--medium">{!$ObjectType.Volunteer_Recurrence_Schedule__c.Label}</h1>
+                    </div>
+                <div class="slds-col slds-no-flex slds-align-middle">
+                        <div class="slds-button-group" role="group">
+                            <apex:commandButton id="cancelBTN" styleClass="slds-button slds-button--neutral" action="{!Cancel}" value="{!$Label.labelButtonCancel}" immediate="true"/>
+                            <apex:commandButton id="saveBTN" styleClass="slds-button slds-button--brand" action="{!Save}" value="{!$Label.labelButtonSave}"/>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!-- PAGE ERRORS -->
+            <c:UTIL_PageMessages />
+            <div class="slds-form--horizontal slds-m-top--large">
+                <h3 class="slds-section-title slds-text-align--left">{!$Label.labelCopySchedule}</h3>
+                <apex:outputPanel id="jrs">
+                    <div class="slds-form-element slds-p-bottom--large slds-p-top--small slds-size--1-of-2">
+                        <apex:actionRegion >
```

Comment:
ok, i figured out why we lose contactId when we do the copy from JRS.  the actionSupport for it is within this actionRegion, so that only the value of slJRS gets shipped to the server.  Thus contact isn't preserved.  But we explicitly do this, so none of the required fields in the lower section generate errors when trying to copy with the JRS.  I think we can live with this issue.  No one has ever complained, and it is less likely, not that the copy JRS is at the top of the page.